src/core/turn-order.js

```diff
@@ -259,12 +259,23 @@ export function UpdateTurnOrderState(G, ctx, turn, endTurnArg) {
   let endPhase = false;
 
   if (endTurnArg && endTurnArg !== true) {
-    if (ctx.playOrder.includes(endTurnArg.next)) {
-      playOrderPos = ctx.playOrder.indexOf(endTurnArg.next);
-      currentPlayer = endTurnArg.next;
-    } else {
+    if ('object' !== typeof endTurnArg) {
       logging.error(`invalid argument to endTurn: ${endTurnArg}`);
     }
+
+    Object.keys(endTurnArg).forEach(arg => {
+      switch (arg) {
+        case 'remove':
+          currentPlayer = getCurrentPlayer(ctx.playOrder, playOrderPos) || '0';
```

Comment:
Would this ever fallback on the `0` at the end of the line?

Reply:
I was basically attempting to do what adjusting playOrderPos in the endTurn method would do (catch out of bounds.) Adjusting that makes this unnecessary I think.